### ISSUE TITLE
Fontmanager not changing the font

### ISSUE THREAD

**Issue by Ducky-bruh** [WilliamRagstad/Font-Manager#48]
I will get to a point where it says it has successfully applied the font I wanted to use, but when I would go launch cs go nothing would change. Does anybody know how to fix this?

**Comment by M1XT3NZ** [comment]
Click on the reset Button and see which folder it takes. Is it the right
one?

If not change it to the correct one. This is probably the problem.

**Comment by Ducky-bruh** [comment]
yeah it always opens to the correct pathing as I checked it by going to my csgo files.

**Comment by M1XT3NZ** [comment]
Hmm,
Well bad timing I’m just about to sleep.

If it say it’s correctly applied it should’ve at least changed the Font
config file from csgo, could you look into that? If anything changes in
there. (Probably not but who knows)

Even if you say it’s a correct path it seems like weird behavior to me.

**Comment by Ducky-bruh** [comment]
nothing changes in my csgo file even though it says it does

**Comment by M1XT3NZ** [comment]
That sounds like it’s taking the wrong csgo folder?

**Comment by Ducky-bruh** [comment]
oh wait i looked again but in the files, it is changing. however it still wont change the font in game.

**Comment by M1XT3NZ** [comment]
delete the file, start csgo and then try again.
maybe that works.

**Comment by Ducky-bruh** [comment]
which file should i delete?

**Comment by M1XT3NZ** [comment]
fonts.conf

in this path.
Counter-Strike Global Offensive\csgo\panorama\fonts

then verrify csgo.

**Comment by Ducky-bruh** [comment]
so i followed those steps but the font is still the default font

**Comment by Ducky-bruh** [comment]
i really just want to be able to use something that's not the default font in csgo.

**Comment by M1XT3NZ** [comment]
okay just tested it myself it works for me,
can you try this?

[fonts.zip](https://github.com/WilliamRagstad/Font-Manager/files/6355369/fonts.zip)

**Comment by Ducky-bruh** [comment]
ok so i deleted the old fonts.conf file and replaced it with this one. then i afterwards applied the font again and even though it is changed in the files, nothing is chaging in game

**Comment by M1XT3NZ** [comment]
oh no you shouldnt have applied the font 
my bad should have said that this is already a font.conf that is changed to a random font

**Comment by Ducky-bruh** [comment]
so after i launch csgo with this new file in, i close it and then apply the font?

**Comment by M1XT3NZ** [comment]
just look if the font changed from the normal csgo font

**Comment by Ducky-bruh** [comment]
nothing changed :(

**Comment by M1XT3NZ** [comment]
that sounds like youre changing files in the wrong folder?

**Comment by Ducky-bruh** [comment]
im changing it in the fonts folder in the panorama folder

**Comment by M1XT3NZ** [comment]
I meant, 
in the right csgo folder :D

**Comment by Ducky-bruh** [comment]
oh yes i went into steam and went to properties-local files-browse local files

**Comment by Ducky-bruh** [comment]
im putting it in the right csgo folder

**Comment by M1XT3NZ** [comment]
im stupid anyway
oops

**Comment by M1XT3NZ** [comment]
[fonts.zip](https://github.com/WilliamRagstad/Font-Manager/files/6355424/fonts.zip)

use this
delete ever yother .ttf file in the csgo folder

**Comment by Ducky-bruh** [comment]
still nothing :(

**Comment by M1XT3NZ** [comment]
you do restart csgo right?

**Comment by Ducky-bruh** [comment]
ofc i restart my game XD

**Comment by M1XT3NZ** [comment]
well then i actually have no idea what is causing this.

sorry.

**Comment by Ducky-bruh** [comment]
yeah i just cant do it at all

even if I try to install it manually via panorama it still dosen't work. i thought font manager could finally let me get a custom font but I guess not rip

**Comment by brycedabull** [comment]
I'm currently having the same problem that @Ducky-bruh was having.  Did you ever find a solution or just give up entirely?

**Comment by brycedabull** [comment]
I'm not sure that this would make any difference but I have my game on an external drive, but all of the font files and everything else are all within the game files so I don't know how that would affect it.

**Comment by M1XT3NZ** [comment]
well did you try all things above?

**Comment by brycedabull** [comment]
I did, tried all of the attached files as well but still couldn't get anything to work.  The fonts.conf file changes every time but nothing changes in game.
